frontend/app/model/Benefits.scala

```diff
@@ -83,43 +92,36 @@ object Benefits {
     "email_updates"
   )
 
-  val friendBenefits = Benefits("Benefits",
-    friendBenefitsList,
-    None,
-    "Become a Friend",
-    """Receive regular updates from the membership community,
-      | book tickets to Guardian Live events and access the Guardian members area""".stripMargin
-  )
-  val supporterBenefits = Benefits(
-    "Benefits",
-    supporterBenefitsList,
-    Some(Pricing(50, 5, Some("1 year membership, 2 months free"))),
-    "Become a Supporter",
-    """Support our journalism and keep theguardian.com free of charge.
-    | Get access to tickets and to the live broadcast of events""".stripMargin
-  )
-  val partnerBenefits = Benefits(
-    "Supporter benefits, plus…",
-    partnerBenefitsList,
-    Some(Pricing(135, 15, Some("1 year membership, 3 months free"))),
-    "Become a Partner",
-    """Get priority booking and 20% discount on Guardian Live,
-      | Guardian Local and most Guardian Masterclasses.""".stripMargin
+  val comparisonBasicList = benefitsFilter(
+    "welcome_pack",
+    "live_stream",
+    "access_tickets",
+    "offers_competitions",
+    "email_updates"
   )
-  val patronBenefits = Benefits(
-    "Partner benefits, plus…",
-    patronBenefitsList,
-    Some(Pricing(540, 60, Some("1 year membership, 3 months free"))),
-    "Become a Patron",
-    """Show deep support for keeping the Guardian free, open and independent.
-    | Get invited to a small number of exclusive, behind-the-scenes functions""".stripMargin
+
+  val comparisonHiglightsList = benefitsFilter(
+    "priority_booking",
+    "discount",
+    "unique_experiences"
   )
 
   def details(tier: Tier) = tier match {
```

Comment:
This looks to be related only to `views`, so it should not be a present in `model`